### PR TITLE
Version bump (0.16.4)

### PR DESCRIPTION
**Pull request by CastagnaIT**
### Check if this PR fulfills these requirements:
* [x] My changes respect the [Kodi add-on development rules](https://kodi.wiki/view/Add-on_rules)
* [x] I have read the [**CONTRIBUTING**](Contributing.md) document.
* [x] I made sure there wasn't another one [Pull Requests](../../pulls) opened for the same update/change
* [x] I have successfully tested my changes locally

#### Types of changes
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Feature change (non-breaking change which change behaviour of an existing functionality)
- [ ] Improvement (non-breaking change which improve functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: addon.xml [hunk 1]
```diff
@@ -55,25 +55,30 @@
     <forum>https://forum.kodi.tv/showthread.php?tid=329767</forum>
     <source>https://github.com/CastagnaIT/plugin.video.netflix</source>
     <news>
+v0.16.4 (2020-01-28)
+-Up Next is now supported only from version 1.1.0 and up
+-Added czech language
+-Added "because you liked" to raccomendations menu
```
Comment (dagwieers) [inline addon.xml]: Typo: recommendations

Comment (CastagnaIT) [reply addon.xml]: haha thanks i have itanialized the word :)

**Comment by dagwieers** [comment]
I think it would be a good practice to attach a ZIP file to the release in GitHub. I know that a branch ZIP file works as well, but a release ZIP file (`make zip`) with a versioned filename is nicer to download and smaller in size.

And it takes about 10 seconds to do :-)

**Comment by CastagnaIT** [comment]
yes but i'm not interested, i would not do even tags,
managing everything is enough and wasting a lot of time

there is the repository, or else
can direct access via Kodi browser (via github weblink) to all releases
or also can roll-back directly from the Kodi "Update" menu to last 2 release, without the need to do any download

**Pull request merged**